pydantic_openapi_helper/helper.py

```diff
@@ -24,6 +24,8 @@ def set_format(p):
     """
     if '$ref' in p:
         return p
+    elif 'anyOf' in p:
+        return p
```

Comment:
Hi @AntoineDao, an object with `anyOf` should not end up here. Am I going o hit this error if I run the code for the Pollination server?

Reply:
Yes. Here is an example of an `anyOf` appearing in the pollination server OpenAPI definition but originally sourced from `queenbee`:

![image](https://user-images.githubusercontent.com/20436635/102108795-44940800-3e2b-11eb-81a7-2155e4c9f45b.png)

Reply:
This was a bug in Pydantic (https://github.com/samuelcolvin/pydantic/issues/1209) which should be fixed now (https://github.com/samuelcolvin/pydantic/pull/1438). I wonder if you are using an older version of Pydantic to generate these docs?